trailblazer/apps/tower/api.py

```diff
@@ -102,6 +103,13 @@ def workflow(self) -> TowerWorkflowResponse:
             url = self.build_url(endpoint=self.workflow_endpoint)
             return TowerWorkflowResponse(**self.send_request(url=url))
 
+    @property
+    def cancel(self) -> None:
+        """Send a POST request to cancel a workflow."""
+        if self.meets_requirements:
```

Comment:
Could this bool be given a more descriptive name? Was confused reading it, but feel free to disregard

Reply:
updated to `send_cancel_request`